jdaviz/configs/imviz/helper.py

```diff
@@ -29,6 +29,40 @@ def default_viewer(self):
         """Default viewer instance. This is typically the first viewer ("imviz-0")."""
         return self._default_viewer
 
+    def create_image_viewer(self):
```

Comment:
Can this have an optional argument `name` that lets the user programatically give it a preferred name?  Or do the viewers need to follow a specific name scheme to work correctly right now?

Reply:
Theoretically, yes. But I need to add a new keyword option here:

https://github.com/spacetelescope/jdaviz/blob/86fce7bc773e79339de4f5f45a1f5f67ce21ede2/jdaviz/app.py#L1168

Reply:
K, if that's going to be a hassle it's not critical if the thing below can be addressed instead.  The key is that it *is* important to be able to create a new viewer and then do something *with the one that was created*, but it's not critical whether that possibility comes from explicitly giving a name (that `get_viewer` then receives) vs returning the name/reference

Reply:
It now returns the viewer instance.

Reply:
It now takes extra argument to let you name it.